conf/simconfig.toml

```diff
@@ -1,8 +1,10 @@
 # PD Simulator Configuration
 
-[tick]
 ## the tick interval when starting PD inside (default: "100ms")
 sim-tick-interval = "100ms"
+total-store = 300
+total-region = 10000000
+case-name = "stable"
```

Comment:
Do other cases work? like scale in/out

Reply:
yes, I have tested in `balance-leader` as well
<img width="1879" alt="image" src="https://github.com/tikv/pd/assets/53859786/2d8b87e3-c00f-44ed-a584-c93d07be189c">

<img width="1855" alt="image" src="https://github.com/tikv/pd/assets/53859786/26501a42-f781-4d35-9f51-609e0b263377">

This case is convenient for add/remove node